### ISSUE TITLE
Issue enabling "qwc-registration-gui"

### ISSUE THREAD

**Issue by meierrom** [qwc-services/qwc-registration-gui#2]
Hi folks,

I'm trying to enable "qwc-registration-gui". The log throws the following:

qwc-registration-gui_1         | sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "qwc-postgis" (172.28.0.17), port 5432 failed: Connection refused
qwc-registration-gui_1         | 	Is the server running on that host and accepting TCP/IP connections?

My config looks like this:

docker-compose.yml
[...]
 qwc-registration-gui:
   image: sourcepole/qwc-registration-gui:v2022.08.22
   environment:
     <<: *qwc-service-variables
     SERVICE_MOUNTPOINT: '/registration'
     DEFAULT_LOCALE: 'en'
     ADMIN_RECIPIENTS: 'admin@example.com'
     MAIL_SUPPRESS_SEND: 'True'
     MAIL_DEFAULT_SENDER: 'from@example.com'
   # ports:
   #   - "127.0.0.1:5032:9090"
   volumes:
     - ./pg_service.conf:/srv/pg_service.conf:ro
[...]

nginx.conf
[...]
    location /registration {
        proxy_pass http://qwc-registration-gui:9090;
    }
[...]

pg_service.conf
[...]
[qwc_configdb]
host=qwc-postgis
port=5432
dbname=qwc_demo
user=qwc_admin
password=qwc_admin
sslmode=disable
[...]

What am I missing?

Kind regards,
Roman

**Comment by tpo** [comment]
To debug this problem:
- from the host where you have installed qwc2 try connecting to 172.28.0.17:5432
- open a shell in the `qwc-registration-gui` container `docker-compose exec qwc-registration-gui` and try connecting from there to 172.28.0.17:5432

**Comment by meierrom** [comment]
Hi!

Thanks for your input!

I restarted my server and the issue no longer appears. Sorry for that.

I think this issue is somehow related to my docker installation. I didn't mention earlier that I started to get a series of warning messages a few days ago upon launching my docker images, which are all of this type:

WARNING: Connection pool is full, discarding connection: localhost

I wonder if this is somehow related to my issue. I wasn't able to fix this yet.

Anyway, according to the docker log the qwc-registration-gui seems to work just fine now.

However, when trying to connect with a browser using localhost:8088/registration, I'm getting 404. The docker log is throwing the following:

qwc-registration-gui_1         | [pid: 7|app: 0|req: 2/2] 172.18.0.18 () {48 vars in 1442 bytes} [Fri Feb 10 09:27:24 2023] GET /registration => generated 232 bytes in 2 msecs (HTTP/1.0 404) 2 headers in 87 bytes (1 switches on core 2)

So, there's still something not working as expected.

Any ideas?

Kind regards,
Roman
